toolkit/tools/imagecustomizerapi/user.go

```diff
@@ -20,6 +20,7 @@ type User struct {
 	SecondaryGroups     []string  `yaml:"secondaryGroups"`
 	StartupCommand      string    `yaml:"startupCommand"`
 	HomeDirectory       string    `yaml:"homeDirectory"`
+	Sudo                bool      `yaml:"sudo"`
```

Comment:
Is this setting specifically enabling *no-password* sudo? If that's the case, then could we make the setting name appropriately specific to that? Given that this is a security-impacting setting, I think we should be as clear as we can about what the image owner is opening up.